src/port/samd.cpp

```diff
@@ -2,12 +2,14 @@
 
 #ifdef ARDUINO_ARCH_SAMD
 
+#if !defined(ARDUINO_API_VERSION)
```

Comment:
Why are those not required anymore? Are they implemented by the api?

Reply:
Yes, the SAMD board release now has USBAPI.h which include routines for USB_SendControl and others.

https://github.com/arduino/ArduinoCore-API/blob/master/api/USBAPI.h

As far as I can tell, the latest SAMD board package releases no longer match github.com/arduino/ArduinoCore-SAMD. It appears the files from ArduinoCore-API are included in the SAMD board packages during the release build procedure.

Reply:
Would you mind finding this out for avr? It would be best to have a solution that works with every board, of course ;-)

Reply:
Any chance to test the AVR code? I'd be happy to merge then ;-)